lib/katello/tasks/upgrades/3.18/add_cvv_export_history_metadata.rb

```diff
@@ -0,0 +1,18 @@
+namespace :katello do
+  namespace :upgrades do
+    namespace '3.18' do
+      task :add_cvv_export_history_metadata => ['environment', 'check_ping'] do
```

Comment:
Is this even necessary ? Export history is new model in 3.18.

Reply:
looks like 3.17 had this model with initial export functionality even (although unlikely to have been used.) So ok with this,